src/include/sof/audio/buffer.h

```diff
@@ -76,6 +77,7 @@ struct comp_buffer {
 	uint32_t pipeline_id;
 	uint32_t caps;
 	uint32_t core;
+	bool inter_core; /* true if connected to a comp from another core */
```

Comment:
Doe it save further logic if we have a flag for source and sink inter_core connections. I can see that using this flag will mean further repeated logic to check if its a source/sink connection ?

Reply:
Sink/source flags don't give us anything, since the same buffer is the source for the one of the components and sink for other.